sdk/src/wallet/core/operations/stronghold_backup/stronghold_snapshot.rs

```diff
@@ -62,7 +59,7 @@ where
     }
 }
 
-pub(crate) async fn restore_from_stronghold_snapshot<S: 'static + SecretManagerConfig>(
+pub(crate) async fn restore_with_stronghold<S: 'static + SecretManagerConfig>(
```

Comment:
Maybe it's just me, but even if private, I find `_with_` to be a bit weird

Reply:
well, the others approved :trollface:

Reply:
Doesn't mean you can't explain that choice :trollface:

Reply:
Not sure but I'd still definitely have the to/from naming instead of with. `restore_fields_from_stronghold_snapshot` or something like that

Reply:
Are you fine with `write_fields_to_stronghold_snapshot` and `read_fields_from_stronghold_snapshot`?